node/config/def.go

```diff
@@ -82,6 +84,22 @@ type SealingConfig struct {
 
 	AlwaysKeepUnsealedCopy bool
 
+	BatchPreCommits     bool
+	MaxPreCommitBatch   int
+	MinPreCommitBatch   int
+	PreCommitBatchWait  Duration
+	PreCommitBatchSlack Duration
```

Comment:
Are there docs describing these things?  I have a general idea what each field does from the names but it's not exactly obvious.